DataMigration/src/main/java/org/sagebionetoworks/dian/datamigration/SynapseUtil.java

```diff
@@ -0,0 +1,970 @@
+                    return site;
+                }
+            }
+            return null;
+        }
+
+        public static Participant findParticipant(String id, Participant[] participants) {
+            for (Participant participant: participants) {
+                if (participant.id != null && participant.id.equals(id)) {
+                    return participant;
+                }
+            }
+            return null;
+        }
+
+        public static ParticipantPhone findParticipantPhone(String id, ParticipantPhone[] phoneList) {
+            for (ParticipantPhone phone: phoneList) {
+                if (phone != null && phone.participant_id.equals(id)) {
+                    return phone;
+                }
+            }
+            return null;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class ParticipantFiles {
+            public File siteLocations;
+            public File participantSiteLocations;
+            public File phone;
+            public File participants;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class SiteLocation {
+            // Table row ID
+            public String id;
+            // Site location name
+            public String name;
+            public String contact_phone;
+            public String contact_email;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class Participant {
+            // Table row ID
+            public String id;
+            // This participant_id is actually the user's ARC ID
+            public String participant_id;
+            // Only used in QA data sets
+            public String name;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class ParticipantSiteLocation {
+            // Table row ID
+            public String id;
+            // This participant is the ParticipantTableRow id field
+            public String participant;
+            // This site_location is the SiteLocationTableRow id field
+            public String site_location;
+        }
+
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public static class ParticipantPhone {
+            // Table row ID
+            public String id;
+            // This participant is the ParticipantTableRow id field
+            public String participant_id;
+            // Internationally formatted phone number, always starting with "+"
+            public String phone;
+        }
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class ParticipantScheduleData {
+        public String participant_id;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class TestSession {
+
+        public TestSession() {
+
+        }
+
+        public String participant_id;
+        public String os_type;
+        public int finished_session; // 1 == finished, 0 == unfinished
```

Comment:
why not use a boolean here instead of an int?

Reply:
This is Happy Medium's data model, I can't change it at this point.